src/pages/contact.jsx

```diff
@@ -38,14 +39,15 @@ const Contact = () => {
       if (xhr.readyState !== XMLHttpRequest.DONE) return
       if (xhr.status === 200) {
         form.reset()
-        setStatus('SUCCESS')
+        setMessage('We received your message. Will get back to you soon.')
+        setShowSuccess(true)
+        setNumber('')
       } else {
-        setStatus('ERROR')
+        setMessage('Unable to send message! Please check you connection.')
```

Comment:
This will push `Connection Error` for every error we will probably get, 
So lets be explicit about response status and handle them individually.